crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -64,12 +95,10 @@ def sync
     redirect_to repositories_path
   end
 
-  #
-  # Activate a single Repository
-  #
-  # Provides the restful api call for
-  # Activate a Repository   /utils/repositories/activate   POST  Creates Repository DataBagItem
-  # required parameters: platform, arch, repo
+  api :POST, "/utils/repositories/activate", "Activate a single repository. Creates a DataBagItem"
+  param :platform, String, desc: "Platform of the repository", required: true
+  param :arch, String, desc: "Architecture of the repository", required: true
+  param :repo, String, desc: "Name of the repository", required: true
```

Comment:
there is no response here in case of success, only 200 OK (like i think in all POST requests in crowbar, except backup create)